lib/ssh_scan/worker.rb

```diff
@@ -27,7 +26,13 @@ def run!
           if response["work"]
             job = response["work"]
             results = perform_work(job)
-            @api_db.add_scan(@worker_id, job["uuid"], results.to_json)
+            if job["db"].eql?("sqlite") && job["save"].eql?("true")
+              api_db = SSHScan::Database::SQLite.new
+              api_db.add_scan(@worker_id, job["uuid"], results)
+            elsif job["save"].eql?("true")
+              api_db = SSHScan::Database::MongoDb.new
+              api_db.add_scan(@worker_id, job["uuid"], results)
+            end
```

Comment:
I feel like the worker shouldn't have any need to know about the DB type.  Isn't that for the API to worry about?